ui/components/app/selected-account/index.scss

```diff
@@ -48,4 +50,10 @@
       background-color: #d9d7da;
     }
   }
+
+  &__copy {
+    height: 13px;
+    margin-left: 3px;
```

Comment:
`margin-inline-start` might be better for RTL

Reply:
@darkwing agreed, I pushed an update to make use of it, thanks!